source/_integrations/jewish_calendar.markdown

```diff
@@ -14,49 +14,34 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The Jewish Calendar (`jewish_calendar`) integration displays a variety of information related to the Jewish Calendar as a variety of sensors.
+The Jewish Calendar (`jewish_calendar`) integration displays various information related to the Jewish Calendar as various sensors.
+{% include integrations/config_flow.md %}
+
 
 ## Configuration
```

Comment:
```suggestion
```
This title can be removed. The config_flow include provides a title.
![image](https://github.com/home-assistant/home-assistant.io/assets/38767475/ffa8dacf-44a2-4c7b-9a71-863093d6b3f6)